tests/__tests__/sitka_module.test.ts

```diff
@@ -102,29 +109,36 @@ describe("SitkaModule", () => {
 
  describe('provideMiddleware adds middleware to Sitka', () => {
 
-   test('middleware provided from a sitka module adds middleware to Sitka', () => {
-      const actual = sitka.createSitkaMeta().middleware[0]
-      expect(actual).toEqual(textModule.historyMiddleware)
-   })
+  test('middleware provided from a sitka module adds middleware to Sitka', () => {
```

Comment:
probably need a test to confirm that no middleware is added to sitka if log is not specified, and history is not specified

Reply:
added